net-libs/libtorrent/libtorrent-0.13.6.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils libtool toolchain-funcs
+
+DESCRIPTION="BitTorrent library written in C++ for *nix"
+HOMEPAGE="https://rakshasa.github.io/rtorrent/"
+SRC_URI="http://rtorrent.net/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE="debug ipv6 ssl"
+
+RDEPEND="
+	>=dev-libs/libsigc++-2.2.2:2
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+#	epatch "${FILESDIR}"/download_constructor.diff
```

Comment:
It's hard to tell definitively if this was fixed upstream. That patch references a bug number on their old bug tracker which was not archived when they moved everything to a new location. Making it hard to see how that issue was ultimately resolved.

With that said, the patch says it's intended as a quick and dirty fix before calling `parse_tracker()`, and that call has since been removed from this bit of code. This makes me think it's safe to remove.

The uncertainty over whether it was actually fixed is likely why it was left in the previous ebuild commented out as well. We're not going to get any more certain in the future so I will remove it.